### ISSUE TITLE
No License?

### ISSUE THREAD

**Issue by noahd1** [ahoward/options#1]
Hi Ara:

Noticed that this library doesn't have a license specified. Can you include one? Also, can you note what the intended license was for this library?

Thanks.

**Comment by zephyr-dev** [comment]
We'd also be interested in knowing this information...

**Comment by jlovejoy** [comment]
please provide license information for this project :)

**Comment by adin-ttc** [comment]
Yes please... most folks in this situation just have a LICENSE file containing "License is the same as the Ruby license"

**Comment by jlovejoy** [comment]
actually, "same as Ruby" is not particularly helpful, considering that Ruby changed from a disjunctive license choice of Ruby or GPLv2 to Ruby or BSD-2-Clause.  So, then one is left wondering if that change also applies to the project in question.  Best case scenario: identify the license (in a brief way) at the top of every source file and provide a full copy in the LICENSE.txt file as you mentioned :)

**Comment by adin-ttc** [comment]
I provided a more complete LICENSE file in my pull as per the suggestion by @jlovejoy.. again simply recommending the use of the Ruby BSD-2-Clause license. Also, I don't believe it's necessary to put the license at the top of each file, but I'm sure it doesn't hurt. If any of you approve please +1 my pull.
